doc/classes/String.xml

```diff
@@ -1028,6 +1028,25 @@
 				Returns a copy of the string with escaped characters replaced by their meanings according to the XML standard.
 			</description>
 		</method>
+		<method name="hex_decode" qualifiers="const">
+			<return type="PackedByteArray"/>
+			<description>
+				Decodes a hexadecimal string as a [PackedByteArray].
+				[codeblocks]
+				[gdscript]
+				var text = "hello world"
+				var buf = text.to_ascii_buffer()
+				var compressed = buf.compress(FileAccess.COMPRESSION_ZSTD).hex_encode()
+				print(compressed.hex_decode().decompress(len(buf), FileAccess.COMPRESSION_ZSTD).get_string_from_ascii())
```

Comment:
Can you please simplify the example? Compression is not directly related to this method.